debian/cephadm.postinst

```diff
@@ -42,15 +41,15 @@ case "$1" in
        fi
 
        # set up (initially empty) .ssh/authorized_keys file
-       if ! test -d /home/cephadm/.ssh; then
-           mkdir /home/cephadm/.ssh
-           chown --reference /home/cephadm /home/cephadm/.ssh
-           chmod 0700 /home/cephadm/.ssh
+       if ! test -d ~cephadm/.ssh; then
```

Comment:
This is a /bin/sh script; is ~username interpolated in /bin/sh?  (yes, on at least Ubuntu, /bin/sh is really dash which does, but I worry about how reliable this is.  I don't think "real" Bourne shell does tilde interpolation.